src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/prisonersearch/PrisonerSearchApiClient.kt

```diff
@@ -22,6 +23,16 @@ class PrisonerSearchApiClient(@Qualifier("prisonerSearchApiWebClient") private v
       ?: emptyList()
   }
 
+  fun getPrisoner(matchPrisonerRequest: MatchPrisonerRequest): List<Prisoner> {
```

Comment:
this should return a single nullable value, `firstOrNull` will retrieve the first item or null if the list is empty

Reply:
okay